### ISSUE TITLE
Separate UI for array of dictionaries

### ISSUE THREAD

**Issue by smaddock** [erikberglund/ProfileCreator#103]
I believe this is a separate issue from #88, if not I apologize.

I noticed this with the PPPC AppleEvents array of dictionaries... it's hard to see what the values are in the current UI:

![screen shot 2018-12-28 at 1 27 34 pm](https://user-images.githubusercontent.com/524425/50527250-711ab200-0aa4-11e9-8977-e7ab31e2323a.png)

But with the current manifest I'm working on it's practically unusable with all the keys:

![screen shot 2018-12-28 at 1 31 27 pm](https://user-images.githubusercontent.com/524425/50527350-edad9080-0aa4-11e9-9e78-73dcaa9d68b3.png)

I'd propose an optional key/flag be available when `pfm_type` is `array` that when true, opens a separate window for *each* array element. So for example, if new key was set for the PPPC AppleEvents, the listbox above would be read-only, but clicking on any element (or the `+`) would open  a new window that looked something like:

![screen shot 2018-12-28 at 1 41 55 pm](https://user-images.githubusercontent.com/524425/50527612-a922f480-0aa6-11e9-82d6-b6c295d305c3.png)

**Comment by erikberglund** [comment]
Yes, this is definitely an issue and I am going to put up a new version within one day, then this feature will be on top of the list for the version after.

**Comment by smaddock** [comment]
Any update on this? Happy to test if needed.
